### PR TITLE
UIORGS-53 Make org status consistent with back-end

### PR DESCRIPTION
**Pull request by AndreiShumski2**
## Purpose
UI operates only with statuses defined in back-end enum from corresponding ticket [MODORGS-25.](https://issues.folio.org/browse/MODORGS-25)

https://issues.folio.org/browse/UIORGS-53
## Approach
In order to ensure consistency, the `organization.status` field should be an enumeration with the following values:
"Pending",
"Inactive",
"Active"

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: translations/ui-organizations/en.json [hunk 1]
```diff
@@ -315,6 +315,9 @@
   "summary.code": "Code",
   "summary.description": "Description",
   "summary.organizationStatus": "Organization status",
+  "summary.organizationStatus.active": "Active",
```
Comment (aliaksei-chumakou) [inline translations/ui-organizations/en.json]: I think it should be just `"organizationStatus.active": "Active",`

File: src/Summary/SummaryForm.js [hunk 2]
```diff
@@ -1,105 +1,189 @@
...
+  Select,
+  TextArea,
+  TextField,
 } from '@folio/stripes/components';
+
 import { Required } from '../Utils/Validate';
 import css from './SummaryForm.css';
 
-class SummaryForm extends React.Component {
+const ORGANIZATION_STATUS = {
```
Comment (aliaksei-chumakou) [inline src/Summary/SummaryForm.js]: IMHO, it's better to pull up into `src/common/constants/organization` or something

**Comment by aliaksei-chumakou** [comment]
@AndreiShumski2 please check status options in Main.js also

File: translations/ui-organizations/en.json (hunk 1 above)
Comment (AndreiShumski2) [reply translations/ui-organizations/en.json]: Updated

File: src/Summary/SummaryForm.js (hunk 2 above)
Comment (AndreiShumski2) [reply src/Summary/SummaryForm.js]: Done

File: src/Accounts/AccountsForm.js [hunk 3]
```diff
@@ -63,82 +93,110 @@ class AccountsForm extends Component {
...
+          <Col
+            xs={12}
+            md={6}
+          >
             <Row>
               <Col xs={12}>
                 <Field
+                  component={Select}
+                  fullWidth
                   label={<FormattedMessage id="ui-organizations.accounts.account.accountStatus" />}
```
Comment (aliaksei-chumakou) [inline src/Accounts/AccountsForm.js]: Are you sure that account status it's the same as organization status?

Comment (AndreiShumski2) [reply src/Accounts/AccountsForm.js]: No, but it has the same statuses.

**Pull request merged**